### PR TITLE
[CDAP-8203] Added secondary call to actions for stream creation

### PR DESCRIPTION
**Pull request by tonybach**
JIRA: https://issues.cask.co/browse/CDAP-8203

"Secondary call to action should have the following logic:
If stream has not being uploaded: [Upload Data to Stream] 
If stream has been successfully uploaded [Query Stream]"

### CODE REVIEW

**Comment by tonybach** [comment]
Build: http://builds.cask.co/browse/CDAP-DRC5571

File: cdap-ui/app/cdap/components/CaskWizards/StreamCreateWithUpload/index.js [hunk 1]
```diff
@@ -69,6 +69,9 @@ export default class StreamCreateWithUploadWizard extends Component {
             // FIXME: I think we can chain this to the next step. TL;DR - will do.
             let url = `/namespaces/${currentNamespace}/streams/${state.general.name}/batch`;
             let fileContents = state.upload.data;
+            if (fileContents) {
+              this.dataUploaded = true;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/CaskWizards/StreamCreateWithUpload/index.js]: wouldn't we be able to find this out from the store? The object that we store in the Store would be an instance of `File` (or we could validate the object api to verify if its a [File](https://developer.mozilla.org/en-US/docs/Web/API/File))

File: cdap-ui/app/cdap/components/EntityCard/FastActions/index.js [hunk 2]
```diff
@@ -92,5 +105,6 @@ FastActions.propTypes = {
   entity: PropTypes.object,
   onUpdate: PropTypes.func,
   className: PropTypes.string,
-  onSuccess: PropTypes.func
+  onSuccess: PropTypes.func,
+  fastActionModalOpened: PropTypes.string
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/EntityCard/FastActions/index.js]: May be `fastActionsToOpen`?. It took me a while to understand what this property does.

**Review by ajainarayanan — COMMENTED** [review]
Minor comments. Otherwise LGTM

**Comment by elfenheart** [comment]
@tonybach try rebasing with release/4.1, and give it a try again. I'm seeing javascript errors right now when I click on upload data to stream CTA

**Comment by ajainarayanan** [comment]
@elfenheart Its failing in NamespaceDropdown `this.apiSubscription.dispose()` is in namespace dropdown which gets initialized when we open the dropdown. If we don't and if it gets unmounted we are disposing it. The problem is we never made a polling call to dispose it. Not sure what changed but this seems to be a different problem

**Review by ajainarayanan — APPROVED** [review]
👍 LGTM

**Pull request merged**